.gitlab-ci.yml

```diff
@@ -81,6 +81,7 @@ Prepare-rhel-internal:
     matrix:
       - RUNNER:
           - aws/rhel-8.4-x86_64
+          - aws/rhel-8.5-x86_64
```

Comment:
This stage is used when testing the latest nightly builds so IMO use only 8.5, not both.

Reply:
Switched to 8.5 only.